src/Microsoft.DotNet.Build.Tasks.net45/project.json

```diff
@@ -8,7 +8,7 @@
     "NuGet.Versioning": "3.6.0-rc-1987",
     "System.Collections.Immutable": "1.1.37",
     "System.Reflection.Metadata": "1.1.0",
-    "System.Runtime.InteropServices.RuntimeInformation": "4.0.0"
+    "System.Runtime.InteropServices.RuntimeInformation": "4.4.0-beta-24806-04"
```

Comment:
Ah! I didn't catch the distinction, that makes sense. Here's old vs. new with what ildasm shows me: https://gist.github.com/dagood/1e9a84cfb4b1ba52d883765e4c8d6468.

That confirms that the api-set DLLs are no longer present and there's the `.module extern kernel32.dll` as expected, but there's also a new dependency:
```
.assembly extern System.Reflection.Extensions
{
  .publickeytoken = (B0 3F 5F 7F 11 D5 0A 3A )                         // .?_....:
  .ver 4:0:0:0
}
```
It's in my GAC, but I don't know if that means it's ok.